cmd/README.md

```diff
@@ -47,6 +47,7 @@ services:
 - what it does: runs the ingestor connected to nats and GraphQL
 - options:
   - nats addr
+  - blob addr
```

Comment:
Before we merge this, should we do a new release as this will cause the docker-compose to break.

Reply:
Released v0.4.0